Assets/Scripts/Cristopher's/ChooseFactionScript2.cs

```diff
@@ -0,0 +1,71 @@
+﻿using UnityEngine;
+using UnityEngine.UI;
+
```

Comment:
Same as above, ChooseFactionScript2 does not tell as much as ChooseSecondFaction, also you can skip adding "Script" in name

Reply:
Done!